### PR TITLE
Added Hindi and Kannada L10N

### PR DESCRIPTION
**Pull request by abartov**

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -759,7 +759,35 @@
...
+       lang_name = 'Kannada',
+       lang_dir = 'ltr',
+       database = 'knwiki_p',
+       wikipedia_domain = 'kn.wikipedia.org',
+       beginners_link = 'https://kn.wikipedia.org/s/sm',
+       beginners_link_title = 'VisualEditorನೊಂದಿಗೆ ಉಲ್ಲೇಖ ಸೇರಿಸುವುದು',
+       reliable_sources_link = 'https://kn.wikipedia.org/s/4u6',
+       lead_section_policy_link = 'https://en.wikipedia.org/wiki/Wikipedia:CITELEAD',
+       lead_section_policy_link_title = 'WP:CITELEAD',
+       citation_needed_category = 'ಉಲ್ಲೇಖಗಳು_ಬೇಕಾಗಿರುವ_ಲೇಖನಗಳು',
```
Comment (eggpi) [inline config.py]: This category and the one for Hindi appear to be empty, no? I'm looking [here](https://hi.wikipedia.org/wiki/%E0%A4%B6%E0%A5%8D%E0%A4%B0%E0%A5%87%E0%A4%A3%E0%A5%80:%E0%A4%B8%E0%A5%8D%E0%A4%B0%E0%A5%8B%E0%A4%A4%E0%A4%B9%E0%A5%80%E0%A4%A8_%E0%A4%95%E0%A4%A5%E0%A4%A8%E0%A5%8B%E0%A4%82_%E0%A4%B5%E0%A4%BE%E0%A4%B2%E0%A5%87_%E0%A4%B8%E0%A4%AD%E0%A5%80_%E0%A4%B2%E0%A5%87%E0%A4%96%27) and Google Translate tells me they're empty.

File: config.py [hunk 2]
```diff
@@ -759,7 +759,35 @@
  	        'Немає АД 2',
         ],
         hidden_category = 'Приховані_категорії',
-    )
+    ),
+
+    hi = dict(
```
Comment (eggpi) [inline config.py]: Are there any citation_needed_templates to add? Or do you just literally use `{{ cn }}` and others from enwiki on these two wikis?

File: config.py (hunk 1 above)
Comment (abartov) [reply config.py]: Oh, you're right.  It looks like the localization was done but not the actual wiki practices the tool assumes, viz. having a template that adds pages to a maintenance category.  I'll discuss this with the communities and get back to you.

**Comment by eggpi** [comment]
Hi, would you still like to pursue this? Did you hear back about the `citation_needed_category` thing?

**Pull request closed without merging**